### PR TITLE
Add vite compilation step & DROP ARMV6

### PR DESCRIPTION
**Pull request by ildyria**
Inspired from the Dockerfile from fly.io

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -74,6 +75,37 @@ RUN \
...
+
+# From our base container created above, we
+# create our final image, adding in static
+# assets that we generated above
+FROM base
+
+# Packages like Laravel Nova may have added assets to the public directory
+# or maybe some custom assets were added manually! Either way, we merge
+# in the assets we generated above rather than overwrite them
+COPY --from=node_modules_go_brrr /app/public /var/www/html/Lychee/public-npm
```
Comment (d7415) [inline Dockerfile:100]:
```suggestion
COPY --from=node_modules_go_brrr --chown=www-data:www-data /app/public /var/www/html/Lychee/public-npm
```
Might mean you can drop the `chown ` later?

Comment (ildyria) [reply Dockerfile:100]: I am not familiar with that part.

Comment (d7415) [reply Dockerfile:100]: Can add in the "may improve later" pile.

**Review by d7415 — APPROVED** [review]
If it works, go for it. I'll see if I can make anything cleaner later.

**Pull request merged**